app/views/layouts/user_mailer.html.erb

```diff
@@ -73,8 +73,8 @@
                                       </tbody>
                                     </table>
                                     <% unless @hide_title %>
-                                      <h4><%= @header || message.subject %>
-                                      </h4>
+                                      <h2><%= @header || message.subject %>
+                                      </h2>
```

Comment:
what if we cleaned up the formatting while we're here?
```suggestion
                                      <h2><%= @header || message.subject %></h2>
```

Reply:
Updated formatting.